homeassistant/components/osoenergy/water_heater.py

```diff
@@ -0,0 +1,304 @@
+"""Support for OSO Energy water heaters."""
```

Comment:
To keep the scope of the PR limited, only add 1 platform at the time (sensor or water_heater)

Reply:
For this PR the sensor has been removed.